gaussfitter/mpfit/mpfit.py

```diff
@@ -407,11 +407,16 @@ def myfunct(p, fjac=None, x=None, y=None, err=None)
    August, 2002.  Mark Rivers
    Converted from Numeric to numpy (Sergey Koposov, July 2008)
 """
-
+from __future__ import division,absolute_import
 import numpy
-import types
-from pyspeckit.spectrum.parinfo import ParinfoList,Parinfo
+from six import integer_types
 from astropy import log
+try:
+    from pyspeckit.spectrum.parinfo import ParinfoList,Parinfo
+    haspyspeckit = True
+except:
+    haspyspeckit = False
```

Comment:
can we make this `HASPYSPECKIT` or `PYSPECKITOK`?  I think there is a convention that global variables like this should be all caps, though I can't point to a specific doc on it...